src/EventStore.Core/TransactionLog/Scavenging/Sqlite/SqliteScavengeBackend.cs

```diff
@@ -18,8 +18,10 @@ public class SqliteScavengeBackend<TStreamId> : SqliteScavengeBackend, IScavenge
 		//    persisted checkpoint.
 		private const string SqliteWalJournalMode = "wal";
 		private const int SqliteNormalSynchronousValue = 1;
+		private const int WalAutocheckpointDisabled = 0;
+
 		private const int DefaultSqliteCacheSize = 2 * 1024 * 1024;
-		private const int SqlitePageSize = 16 * 1024;
+		private const int SqlitePageSize = 64 * 1024;
```

Comment:
on ZFS, the default logical block size is 128 kb (recordsize parameter)

- could it be the reason why better performance was observed with 64kb on cloud? (and would 128kb work even better?)
- can it result in worse performance on other file systems when doing many updates? (with the standard 4KB logical block sizes)

Reply:
yeah fair questions, decided to make this configurable with a default of 16k as it was before

Reply:
thanks!